src/sentry/static/sentry/app/components/projects/missingProjectMembership.tsx

```diff
@@ -116,7 +116,15 @@ class MissingProjectMembership extends React.Component<Props, State> {
     const teams = this.state.project?.teams ?? [];
     teams.forEach(({slug}) => {
       const team = TeamStore.getBySlug(slug);
-      team?.isPending ? pending.push(team!.slug) : request.push(team!.slug);
+      if (!team) {
+        addErrorMessage(
+          t(
+            'There was an error while trying to request a team associated with this project.'
+          )
+        );
+      } else {
+        team?.isPending ? pending.push(team!.slug) : request.push(team!.slug);
```

Comment:
Now that we check that `team` exists, I don't think we need the optional chaining operator (`team?.`) or the null assertion (`team!`).